blueprints/hpc-cluster-small.yaml

```diff
@@ -28,6 +28,17 @@ deployment_groups:
         id: network1
         use: [project1]
 
+      - source: ./modules/billing
+        kind: terraform
+        id: billing
+        settings:
+          display_name: example-alert"
+          amount: 1000
+          currency_code: USD
```

Comment:
This should be a variable.

Reply:
all of these are already variables...
does the CB call and use this blueprint as it is ?